siege_utilities/identifiers/namespaces.py

```
@@ -0,0 +1,70 @@
+
+    Args:
+        seed: A non-empty string. Convention: lowercase DNS-style name.
+
+    Returns:
+        The UUID5 derived from ``NAMESPACE_URL`` and ``seed``.
+
+    Raises:
+        ValueError: if ``seed`` is empty.
+    """
+    if not seed:
+        raise ValueError("root seed must be non-empty")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)